lib/private/docker.ts

```diff
@@ -249,7 +259,7 @@ export class DockerFactory {
    * Gets a Docker instance for building images.
    */
   public async forBuild(options: DockerFactoryOptions): Promise<Docker> {
-    const docker = new Docker(options.logger);
+    const docker = new Docker(options.eventEmitter);
```

Comment:
In your mind, is this code churn worth it?

Reply:
🤷‍♂️ I didn't like the name logger since log to me means "print to stdout/stderr" and that's not always true anymore

Reply:
I guess this us ascribing different meanings to words. You interpret "logger" to mean "the thing that prints messages to stdio" and I interpret logger to mean "the thing that receives messages and does *something* with them". Could be printing, could be writing to a file, could be sending over HTTP to a syslog daemon or to CloudWatch.

Reply:
Strictly speaking, the thing you're passing right now is not an event **emitter**, but rather an event **receiver** 😛